common/src/main/java/net/william278/husksync/database/MySqlDatabase.java

```diff
@@ -33,6 +33,10 @@
 import java.util.logging.Level;
 
 public class MySqlDatabase extends Database {
+    /**
```

Comment:
style nit: space before javadoc comment for member variable after class declaration